modules/property-details/index.js

```diff
@@ -1,106 +1,193 @@
+};
+
+const getSincePurchase = (recentValuation, originalPurchasePrice) => {
+  return recentValuation.amount - originalPurchasePrice;
+};
+
+const getSincePurchasePercentage = (recentValuation, originalPurchasePrice) => {
+  return (getSincePurchase(recentValuation, originalPurchasePrice) / originalPurchasePrice) * 100
+};
+
+const renderCustomAccountListItem = (originalPurchasePriceDate, originalPriceValue) => {
+  return (<AccountList>
+    <AccountListItem>
+      <>
+        <InfoText>{`Purchased for`}&nbsp;</InfoText>
+        <InfoText><strong>{originalPriceValue}</strong></InfoText>
+        <InfoText>&nbsp;{`in ${MONTH_NAMES[new Date(originalPurchasePriceDate).getMonth()-1]} ${new Date(originalPurchasePriceDate).getFullYear()}`}</InfoText>
+      </>
+    </AccountListItem>
+    <AccountListItem><InfoText>Since Purchase</InfoText></AccountListItem>
+    <AccountListItem><InfoText>Annual Appreciation</InfoText></AccountListItem>
+  </AccountList>)
 };
 
+//this would be a more agnostic approach component in terms of business logic :)
+const TitledList = ({labels, title, headLineInfo = null, handleClick = () => null}) => (
+  <AccountSection>
+    <AccountLabel>{title} </AccountLabel>
+    {headLineInfo && <AccountHeadline>
+      {headLineInfo}
+    </AccountHeadline>}
+    <RowContainer onClick={handleClick}>
+      <AccountList>
+        {labels.map((label, key) => <AccountListItem key={key}><InfoText>{label}</InfoText></AccountListItem>)}
+      </AccountList>
+    </RowContainer>
+  </AccountSection>
+
+);
+//TO.DO: Create the business component that hides agnostic component also for product details and for product estimated value.
+const MortgageSection = ({currentBalance, name, handleClickMortgage}) => (
+  <TitledList title='Mortgage' labels={[currentBalance, name]} handleClick={handleClickMortgage}/>
+);
+
+const ValuationChangeSection = (account)=> {
+  const {recentValuation, originalPurchasePrice, originalPurchasePriceDate} = account;
+  const yearsSincePurchase = getYearsSincePurchase(originalPurchasePriceDate);
+
+  const sincePurchasePercentage = getSincePurchasePercentage(recentValuation, originalPurchasePrice);
+
+  const originalPriceValue = new Intl.NumberFormat("en-GB", {
+    style: "currency",
+    currency: "GBP",
+    maximumSignificantDigits: 3
+  }).format(
+    Math.abs(originalPurchasePrice)
+  )
+
+  const sincePurchaseValue = new Intl.NumberFormat("en-GB", {
+    style: "currency",
+    currency: "GBP",
+    maximumSignificantDigits: 3
+  }).format(
+    Math.abs(getSincePurchase(recentValuation, originalPurchasePrice))
+  )
+
+
+  const sincePurchaseInfo = `${sincePurchaseValue} (${sincePurchasePercentage}%)`;
+  return (
+      <AccountSection>
+        <AccountLabel>Valuation change</AccountLabel>
+        <SectionWrapper>
+          {renderCustomAccountListItem(originalPurchasePriceDate, originalPriceValue)}
+          <div>
```

Comment:
I just used this approach in order not to alterate so much the initial components.